darwin/datatypes.py

```diff
@@ -360,6 +360,12 @@ class Slot:
     #: Metadata of the slot
     metadata: Optional[Dict[str, UnknownType]] = None
 
+    #: Frame Manifest for video slots
+    frame_manifest: Optional[List[Dict[str, UnknownType]]] = None
```

Comment:
Completely optional change, but `Dict[str, UnknownType]` is interchangable with `JSONType`